### ISSUE TITLE
Upgrade to latests matomo-org/device-detector version

### ISSUE THREAD

**Issue by jmaitrehenry** [etienne-martin/device-detector-js#134]
It's would be a great move to update this lib to the latest version of matomo-org/device-detector (v4.3.1).

This should fix issues:
- #129 (v4.3.1)
- #114 (v4.1.0)
- #109 (v4.1.0)
- #137

**Comment by dseeker** [comment]
I'm having problems with latest FireTV 4k Max detected as a phone, seems the latest version has a fix for it, is this close to release?

**Comment by etienne-martin** [comment]
Are you getting the correct result when using this site https://devicedetector.lw1.at ?

**Comment by dseeker** [comment]
no, I've copied the Amazon Fire TV 4k max user agent from the web app and using an emulator it's detected as a phone
![image](https://user-images.githubusercontent.com/11655138/147140629-630236c5-28e6-45be-a144-9b94b85cc243.png)

here's the commit that fixes this issue which many of our customers are complaining about
https://github.com/matomo-org/device-detector/commit/5d1fa4033214ce22806c5ff7f88589ab3c0c21d4

here's the user agent so you can test
`Mozilla/5.0 (Linux; Android 9; AFTSS Build/PS7234; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/87.0.4280.101 Mobile Safari/537.36 cordova-amazon-fireos/3.4.0 AmazonWebAppPlatform/3.4.0;2.0`

**Comment by etienne-martin** [comment]
The latest version of matomo works fine with this user-agent: 

```
Gibbon/2020.2.2.0/2020.2.2.0: Netflix/2020.2.2.0 (DEVTYPE=NFANDROID2-PRV-FIRETVSTICK2018; CERTVER=0)
```

Found it in their [test fixture](https://github.com/matomo-org/device-detector/commit/5d1fa4033214ce22806c5ff7f88589ab3c0c21d4#diff-761900e532f540b0cc87f155c854aff04ed979558333c796b1d95530ac8eba97R2571).

I haven't looked at the new regex introduced by matomo-org/device-detector@5d1fa40 to see what could cause yours to fail. 

Even if we were to update the library with what's currently upstream, the user-agent you provided would still fail so you can report the issue at https://github.com/matomo-org/device-detector/issues if you think there's something wrong with the detection of that user-agent.

Once the issue is fixed upstream, we'll be able to pull these changes into the library. No guarantee that this will be done quickly however.

**Comment by sanchezzzhak** [comment]
`Mozilla/5.0 (Linux; Android 9; AFTKA Build/PS7255; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/94.0.4606.118 Mobile Safari/537.36 cordova-amazon-fireos/3.4.0 AmazonWebAppPlatform/3.4.0;2.0`
added to https://github.com/matomo-org/device-detector/pull/6969
